README.md

```diff
@@ -68,6 +69,27 @@ The webapp/webpage is atm just to show the QRcode with out of band invitation fo
 
 **Open the webpage for develop** - open> `file:///.../webapp/index-fastopt.html`
 
+### Configure the Mediator
+
+The default configuration is set up [application.conf](/mediator/src/main/resources/application.conf).
+So in order to configure the mediator for your needs.
+You can either change the default configuration, or you can se up environment variables that overrides the defaults:
+
+For set up the mediator identity:
+- `KEY_AGREEMENT_D` - is the key agreement private key (MUST be a X25519 OKP key type).
+- `KEY_AGREEMENT_X` - is the key agreement public key (MUST be a X25519 OKP key type).
+- `KEY_AUTHENTICATION_D` - is the key authentication private key (MUST be an Ed25519 OKP key type).
+- `KEY_AUTHENTICATION_X` - is the key authentication public key (MUST be an Ed25519 OKP key type).
+- `SERVICE_ENDPOINT` - is the endpoint of the mediator. Where the mediator will be listening to incoming DID Comm messages.
+
+For set up the mediator storage (MongoDB):
+- `MONGODB_PROTOCOL` - is the protocol type used by mongo.
+- `MONGODB_HOST` - is the endpoint where the MongoDB will be listening.
+- `MONGODB_PORT` - is the endpoint's port where the MongoDB will be listening.
+- `MONGODB_USER` - is the user name used by the Mediator service to connect to the database.
+- `MONGODB_PASSWORD` - is the password used by the Mediator service to connect to the database.
+- `MONGODB_DB_NAME` - is the name of the database used by the Mediator.
+
 ## Run
 
 This DIDComm Mediator is composed of two elements, a backend service and a the database.
```

Comment:
Please, check this file for typos like that: `a the database`, it could be nice to use something like https://app.grammarly.com/ to fix these errors automatically.

Reply:
We already give all the defaults here `The default configuration is set up [application.conf](/mediator/src/main/resources/application.conf).`

Although I was thinking that maybe we should not give them the private keys for the DID because almost everyone will use it...

Reply:
I was using Grammarly very often before. I will check